pkg/pillar/testprograms/testverifier/testverifier.go

```diff
@@ -17,6 +17,7 @@ package main
 
 import (
 	"crypto"
+	//nolint:staticcheck
```

Comment:
Actually testverifier has not been used since the original implementation, so we might as well delete it.

Reply:
I removed it in a separate PR since this yetus complaint started showing up in all PRs.

Reply:
Thank you!